src/styles/blocks/catalog.scss

```
@@ -18,6 +20,14 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: color $transitionSpeed, transform $transitionSpeed;
+
+    &:hover {
+      color: #34568b;
```

don't remember create variables for colors
```suggestion
      color: $c-someColor;
```